li-utils/src/main/java/com/linkedin/common/urn/DatasetFieldUrn.java

```diff
@@ -0,0 +1,101 @@
+package com.linkedin.common.urn;
+
+import com.linkedin.common.FabricType;
+import com.linkedin.data.template.Custom;
+import com.linkedin.data.template.DirectCoercer;
+import com.linkedin.data.template.TemplateOutputCastException;
+import java.net.URISyntaxException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+/**
+ * Standardized dataset field information identifier
+ */
+public class DatasetFieldUrn extends Urn {
+
+  // uniquely identifies urn's key type
+  public static final String ENTITY_TYPE = "datasetField";
+
+  // urn pattern
+  private static final Pattern DATASET_FIELD_URN_PATTERN = Pattern.compile(
+      "urn:li:datasetField:\\(urn:li:dataset:\\(urn:li:dataPlatform:(?<dataPlatform>.+),(?<datasetName>.+),(?<fabric>.+)\\),(?<field>.+)\\)");
+
+  /**
+   * Dataset urn of the datasetFieldUrn
+   */
+  private final DatasetUrn _dataset;
+
+  /**
+   * Field of datasetFieldUrn
+   */
+  private final String _field;
```

Comment:
you might want to call it fieldPathSpec or fieldPath to be explicit.

Reply:
Agree.